pymeasure/instruments/rohdeschwarz/fsl.py

```diff
@@ -150,6 +151,36 @@ def read_trace(self, n_trace=1):
             x = np.linspace(self.freq_start, self.freq_stop, len(y))
 
         return np.array([x, y])
+'''
+
+    def read_trace(self, n_trace=1):
+
+        """
+        Read trace data from the active channel.
+
+        :param n_trace: The trace number (1-6). Default is 1.
+        :return: 2d numpy array of the trace data, [[frequency], [amplitude]].
+        """
+
+        trace_data = np.array(self.values(f"TRAC? TRACE{n_trace}"))
+        # dict = self.available_channels
```

Comment:
Why just comment out? If it is not used then there is no need to keep it as a comment.

Reply:
Resolved via [697127a](https://github.com/bleykauf/pymeasure/pull/4/commits/697127a14c501fae622095b3e2ec04cabfc76fdd)